css-two-column-layout/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="py-7 text-center">
+      <h1>Scale, Speed, Agility</h1>
+      <p class="paraheader">We've dedicated thousands of man hours to making healthcare institutions
+        in the Helium network run 10x more efficiently and deliver an unparralled quality of care.
+      </p>
+    </div>
+    <div class="info-card-grid">
+      <div class="row">
+        <div class="info-card-image-column">
```

Comment:
`column` classes should be named after their size, not their content. Make sure you address this on any other `column` classes in this document as well.